test/mint/Mint.behavior.ts

```diff
@@ -104,7 +104,7 @@ export function shouldBehaveLikeHumanboundMint(): void {
             beforeEach("construct ethereum access token", async function () {
               this.params = [this.signers.user0.address, tokenId];
               this.value = {
-                expiry: BigNumber.from(Math.floor(new Date().getTime() / 1000) + 200),
+                expiry: BigNumber.from(Math.floor(new Date().getTime() / 1000) + 2000),
```

Comment:
Why is this change needed?

Reply:
So for some reason it looks like the EAT is being constructed at a time much before it seems to be used so the addition of the Approve tests seems to cause the EAT here to be expired by the time it gets consumed. Prolonging the expiry date seems to solve this problem.

Reply:
lol ok I see

Reply:
Regarding tasks. I'm personally of the view that tasks are more utilities/helpers and should remain outside of feature PRs so these can remain focused on feature improvements and maybe we can bundle utilities and other helpers in a separate PR later.